chaincode/ledger.go

```diff
@@ -133,6 +138,7 @@ type CompositeTraintuple struct {
 	InTrunkModel  string                      `json:"inTrunkModel"`
 	OutHeadModel  CompositeTraintupleOutModel `json:"outHeadModel"`
 	OutTrunkModel CompositeTraintupleOutModel `json:"outTrunkModel"`
+	Perf          float32                     `json:"perf"`
```

Comment:
@inalgnu I just noticed this line, is this addition intentional?

Reply:
nope this seems to be due to a bad rebase :-/

Reply:
https://github.com/SubstraFoundation/substra-chaincode/compare/master...AurelienGasser-patch-2?quick_pull=1